content/community/tools-data-fetching.md

```diff
@@ -1,12 +1,12 @@
 ---
 id: data-fetching
-title: Data Fetching
+title: جلب البيانات
```

Comment:
This translation is the best translation for 'fetch' I've ever read :) we should add it to the Glossary and agree on using it